Controller/RegistrationFOSUser1Controller.php

```diff
@@ -64,13 +65,14 @@ public function registerAction()
                     $this->get('session')->remove('sonata_basket_delivery_redirect');
                     $url = $this->generateUrl($route);
                 } else {
-                    $url = $this->get('session')->get(
-                        'sonata_user_redirect_url',
-                        $this->generateUrl('sonata_user_profile_show')
```

Comment:
You are right!

Just for anyone that was thinking like me: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/HttpFoundation/Session/Attribute/AttributeBag.php#L83